src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,15 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (number) => {
+  const catYears = (number - 2) * 4 + 24;
```

Comment:
There is nothing wrong with using a few if statements in cases like this. Generally nested ternaries are a bad thing, as they (usually) take longer to understand than just breaking code up into multiple ifs instead. Theres a lot of debate around this but in my experience they are almost never used for this reason. Extra credit though for trying it out regardless :)